src/components/graphs/bargraph/HorizontalBar.js

```diff
@@ -16,7 +16,7 @@ const Horizontalbar = () => {
 
   const data = {
     labels: [
-      'Rank 2 - Empty-hand',
```

Comment:
I don't believe we can change this, I may be wrong because they're labels

Reply:
That updates the graph display text, and it still seems to work. I only changed the display text to read "Empty Hand" and left the handler/variable code as "empty-hand."